patricksimpson/gem-randomizer
# SC-460: Redirect http to https

## Description
Redirect http://gems.seesparkbox.com to https://gems.seesparkbox.com/
In order to get on the Chrome HSTS preload list, our subdomains need to have HTTPS and redirect HTTP to HTTPS. 

## To Test
1. Make sure all PR Checks have passed (CircleCI, Code Climate, Snyk, etc).
1. Pull down this branch.
1. Confirm all tests pass: `npm run test` & `npm run test:a11y`
1. Navigate to http://localhost:9292 and confirm that is redirects to https://localhost:9292

Once this checks out, I will update the redirect on this line to 
```use Rack::Rewrite do
  r301 %r{.*}, 'https://gems.seesparkbox.com/', :scheme => 'http'
end```
